ports/nrf/boards/pitaya_go/mpconfigboard.mk

```diff
@@ -0,0 +1,10 @@
+USB_VID = 0x239A
+USB_PID = 0x802A
```

Comment:
Do not recycle VID/PID, they must be unique for each device!

Reply:
OK. As Makerdiary doesn't have a VID, [maybe use a random generated PID with NordicSemi's VID](https://devzone.nordicsemi.com/f/nordic-q-a/50638/usb-pid-for-nrf52840)

Reply:
hi, as you make many products with USB, please purchase a VID!
https://store.makerdiary.com/

Reply:
can not afford the licensing fee yet